### PR TITLE
blog: May 2018 newsletter

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Should go out today, ideally. Let me know if I missed anything (although it's already pretty busy :P)

### CODE REVIEW

**Review by orrabin — APPROVED** [review]

File: _posts/2018-05-31-foreman-community-newsletter---may-2018.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+- newsletter
+- community
+excerpt: |
+  Foreman 1.17.1, Katello 3.5.2, GDPR, Summit, and all the usual goodies.
+---
+
+### Foreman Privacy Policy created
+
+Prompted by the incoming EU GDPR legislation, the Foreman project has finally
+written down it's [privacy policy](/privacy.html), and you can find it in the
```
Comment (mmoll) [inline _posts/2018-05-31-foreman-community-newsletter---may-2018.md]: its

File: _posts/2018-05-31-foreman-community-newsletter---may-2018.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+In just 2 weeks, Foreman will be taking part in the
+[OpenSourceCamp.de](https://opensourcecamp.de) event. We're teaming up with
+Graylog to bring you a day of infrastructure management talks, and it's right
+after [OSDC](https://osdc.de) so you can get two conferences for your travel
+budget! We've got a great speaker line-up for you, and a booth where you can
+come chat with us in the breaks. See you there!
+
+### Foreman 1.17.1 released
+
+Just 2 days ago Ondrej released 1.17.1 containing a bunch of important fixes
```
Comment (mmoll) [inline _posts/2018-05-31-foreman-community-newsletter---may-2018.md]: Ondřej

File: _posts/2018-05-31-foreman-community-newsletter---may-2018.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+You can find details on the bugs fixed in the [release announcement][1_17_1],
+and thank you to everyone that reported the issues and worked on the fixes. As
+always bugs can be reported on [issue tracker][issues] - thanks!
+
+### Katello 3.5.2 is out
+
+The next release in the Katello 3.5 stream was released earlier this month,
+containing 20 fixes in Katello alone, and more in subpackages. This is suitable
+for Foreman 1.16, and you can check the [announcement][k_3_5_2] For more
```
Comment (mmoll) [inline _posts/2018-05-31-foreman-community-newsletter---may-2018.md]: for more

File: _posts/2018-05-31-foreman-community-newsletter---may-2018.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+eyeballs. Here's this month's selection:
+
+  * [Network Management](https://community.theforeman.org/t/foreman-and-network-subnet-management/9787)
+  * [Webpack for plugins](https://community.theforeman.org/t/webpacking-plugins-lets-do-it-right/9558)
+  * [Changing templates menu](https://community.theforeman.org/t/ux-rfc-move-templates-to-the-top-menu-or-provision-menu/9745)
+  * [Denormalizing the fact tables](https://community.theforeman.org/t/rfc-denormalize-fact-tables/9611)
+
+### [Upcoming Events](https://community.theforeman.org/c/events/l/calendar)
+
+The Foreman Community has now fully embraced Discourse as the source of it's
```
Comment (mmoll) [inline _posts/2018-05-31-foreman-community-newsletter---may-2018.md]: its

**Comment by mmoll** [comment]
I do have some nitpicks :)

**Comment by GregSutcliffe** [comment]
@mmol - fixxorred, oh master of the nits! I bow before your supreme pedantry :grin: Can haz ack?

**Comment by mmoll** [comment]
looking good now, merged, thanks @GregSutcliffe!

**Pull request merged**